alcs-frontend/src/app/features/notice-of-intent/decision/decision-v2/decision-input/decision-conditions/decision-condition/decision-condition.component.ts

```diff
@@ -56,10 +56,10 @@ export class DecisionConditionComponent implements OnInit, OnChanges {
 
   form = new FormGroup({
     securityAmount: this.securityAmount,
-    singleDate: this.singleDate,
```

Comment:
And this one

Reply:
Same here:
form = new FormGroup({
    securityAmount: this.securityAmount,
    administrativeFee: this.administrativeFee,
    description: this.description,
    componentsToCondition: this.componentsToCondition,
    singleDate: this.singleDate,
  });